bot/exts/evergreen/githubinfo.py

```diff
@@ -74,24 +83,71 @@ async def get_github_info(self, ctx: commands.Context, username: Optional[str])
 
                 embed.add_field(name="Followers",
                                 value=f"[{user_data['followers']}]({user_data['html_url']}?tab=followers)")
-                embed.add_field(name="\u200b", value="\u200b")
                 embed.add_field(name="Following",
                                 value=f"[{user_data['following']}]({user_data['html_url']}?tab=following)")
 
             embed.add_field(name="Public repos",
                             value=f"[{user_data['public_repos']}]({user_data['html_url']}?tab=repositories)")
-            embed.add_field(name="\u200b", value="\u200b")
 
             if user_data['type'] == "User":
                 embed.add_field(name="Gists", value=f"[{gists}](https://gist.github.com/{username})")
 
                 embed.add_field(name=f"Organization{'s' if len(orgs)!=1 else ''}",
                                 value=orgs_to_add if orgs else "No organizations")
-                embed.add_field(name="\u200b", value="\u200b")
             embed.add_field(name="Website", value=blog)
 
         await ctx.send(embed=embed)
 
+    @github_group.command(name='repo', aliases=('repository',))
+    async def github_repo_info(self, ctx: commands.Context, repo: Optional[str]) -> None:
+        """
+        Fetches a repositories's GitHub information. Repository should look like `user/reponame`.
+
+        Repository is optional and sends the help command if not specified.
+        """
+        if repo is None:
+            await invoke_help_command(ctx)
+            ctx.command.reset_cooldown()
+            return
+        async with ctx.typing():
+            repo_data = await self.fetch_data(f"{GITHUB_API_URL}/repos/{repo}")
+
+            # There won't be a message key if this repo exists
+            if repo_data.get('message') is not None:
+                await ctx.send(embed=discord.Embed(title=random.choice(NEGATIVE_REPLIES),
+                                                   description="The requested repository was not found.",
+                                                   colour=discord.Colour.red()))
+                return
+
+        repo_owner = repo_data['owner']
+
+        parent = repo_data.get('parent')
+
+        embed = discord.Embed(title=f"{repo_data['name']}",
```

Comment:
This is against the style guide, each argument should be on a separate line.
example:
```py
embed = (
    title="...",
    description="...",
)
```
Instead of:
```py
embed = (title="...",
    description="...."
)
```